pkgs/leak_tracker_flutter_testing/README.md

```diff
@@ -0,0 +1,22 @@
+[![pub package](https://img.shields.io/pub/v/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing)
+[![package publisher](https://img.shields.io/pub/publisher/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing/publisher)
+
+Coming soon! See https://github.com/flutter/devtools/issues/3951.
+
+The text below is under construction.
+
+## What is this?
```

Comment:
I'd also answer the question about why this package is separate from the non `*_testing` version.